rfc/text/advanced/ap_10_rpc_progressive_calls.md

```diff
@@ -115,6 +115,17 @@ If the *Callee* supports progressive calls, the *Dealer* shall forward the *Call
 A call invocation MUST *always* end in a *normal* `CALL` without the `"progress": true` option, or explicitly 
 set `"progress": false` which is the default.
 
+**Request Id Verification**
+
+When the *Progressive Calls* feature is supported, a *Caller* MAY use a prior request ID in a `CALL` message to
```

Comment:
But there **is** sequential request ID verification mandated for `CALL` messages in this section https://wamp-proto.org/wamp_latest_ietf.html#name-protocol-errors . To avoid confusion, I'll rewrite this from the point of view of the Dealer performing the verification.

`CALL` is unique among the request type of messages in that the request ID can either be correlated to an existing active request, or be a new request.

I should add a note in that Protocol Errors section that there are exemptions for progressive calls.